### PR TITLE
WIP/Sigmoid accel curve to avoid jerky moves in slow speeds

### PR DESCRIPTION
**Pull request by ankostis**
- This is just the implementation, no docs/diagrams updated yet.
- The current defaults (max-factor=4) should work with DPI=200 (i think).
  These may not be optimal and we change them before accepting it.
-  A python notebook (vscode) is includnd to design the curves needed to experiment, or else try [Wolfram Alfa](https://www.wolframalpha.com/input?i=plot+1%2B%28c-1%29%2F%281+%2B+e%5E%28-%28x-b%29*a%29%29+with+a%3D0.9+with+b%3D5+with+c%3D4+from+x%3D-0.1+to+10+from+y%3D-0.5+to+4.5).
- Note that i prefer to [rewrite git-history](https://sethrobertson.github.io/GitBestPractices/#sausage), particularly when commit messages begin with WIP (Work In Progress) or the bug was blatant.
 
image

### CODE REVIEW

**Comment by ankostis** [comment]
For this curve i suggest these steppings:
```c
#        define MACCEL_STEEPNESS_STEP  from 0.01 --> 0.1
#        define MACCEL_OFFSET_STEP     from 0.1  --> 0.5
#        define MACCEL_LIMIT_STEP      from 0.1  --> 1.0
```

**Comment by ankostis** [comment]
TL;DR, closing this,  superseded by dev branch with a much better continues curve.

 [Wimads discovered](https://discord.com/channels/681309835135811648/1203417572742135808/1208883598078255136) the [generalized logistic function](https://en.wikipedia.org/wiki/generalised_logistic_function) which has an independent  param for the low ends where accuracy is an issue, without the complications of mixing 2 pure sigmoids.

Regarding the python code also  contained in this PR, Wimads also discovered [the Desmos online graphing calculator](https://discord.com/channels/681309835135811648/1203417572742135808/1208864467350454294) which can replace my interactive notebook for this and many other functions.
